### PR TITLE
guide: consolidate Exp Sharing intro

### PR DESCRIPTION
**Pull request by jorgeorpinel**
The idea behind this "consolidate" series of PRs (more to come) is to

a) summarize the explanations in this guide (without removing necessary info) and
b) to remove/bring details from the command reference (and link) per task #2 in https://github.com/iterative/dvc.org/issues/2548.

---

Follow up to https://github.com/iterative/dvc.org/pull/2618

> Cascaded into #2654

- [ ] Next chunk: https://github.com/iterative/dvc.org/pull/2719

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
@iesahin the main changes to review here are in content/docs/user-guide/experiment-management/sharing-experiments.md . I thought we could do some small follow-up PRs like these to work on summarizing content. Feel free to contribute more changes or suggest alternatives beyond just reviewing. Thanks!

File: content/docs/user-guide/experiment-management/sharing-experiments.md [hunk 1]
```diff
@@ -1,37 +1,27 @@
...
+[Git remotes](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes)
+are distributed copies of the Git repository, hosted for example on GitHub or
+GitLab. Small files like experimental code and
+[DVC metafiles](/doc/user-guide/project-structure) files will go there.
+[DVC remotes](/doc/command-reference/remote) on the other hand are data storage
+locations (e.g. Amazon S3 or Google Drive). You can use them to back up and
+[share data](/doc/use-cases/sharing-data-and-model-files) files and directories
+that don't fit inside Git repos.
+
+> See this [Git remotes guide] and `dvc remote add` for ifo. on setting them up.
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/sharing-experiments.md:13]: Oops typo (fixing)
```suggestion
> See this [Git remotes guide] and `dvc remote add` for info. on setting them up.
```

File: content/docs/user-guide/experiment-management/index.md [hunk 2]
```diff
@@ -47,12 +47,14 @@ The `dvc experiments` features are designed to support these main approaches:
    them in batches.
 1. Make experiments [persistent] by committing them to your
    <abbr>repository</abbr> history.
+1. Easily [share and isolate] experiments using Git and DVC remotes.
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/index.md:50]: This resolves task #2 from https://github.com/iterative/dvc.org/issues/2687 BTW/

Comment (iesahin) [reply content/docs/user-guide/experiment-management/index.md:50]: "Isolate and share" or "share and isolate"?

**Review by iesahin — APPROVED** [review]
LGTM. Thank you Jorge.

File: content/docs/user-guide/experiment-management/sharing-experiments.md [hunk 3]
```diff
@@ -1,37 +1,27 @@
...
   https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
 
-Normally, there should already be a Git remote called `origin` when you clone a
-repo. Use `git remote -v` to list your Git remotes:
+You can list your remotes with `git remote -v` and `dvc remote list`:
 
 ```dvc
 $ git remote -v
 origin  https://github.com/iterative/get-started-experiments (fetch)
 origin  https://github.com/iterative/get-started-experiments (push)
```
Comment (daavoo) [inline content/docs/user-guide/experiment-management/sharing-experiments.md:23]: I think that with those remotes `dvc exp push` won't actually work, it should be:

```suggestion
origin  git@github.com:iterative/get-started-experiments.git (fetch)
origin  git@github.com:iterative/get-started-experiments.git (push)
```

**Review by daavoo — COMMENTED** [review]
I think that we should point out that `dvc exp push`/`dvc exp pull` only work with SSH:

> using git over ssh with the correct SSH keys configured is the only supported method for authenticated git push in dvc exp push
(from https://iterativeai.slack.com/archives/CB41NAL8H/p1628743292077800?thread_ts=1628690244.073500&cid=CB41NAL8H)

File: content/docs/user-guide/experiment-management/index.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/experiment-management/index.md:50]: I like share first since it's the title of the page linked 🙂

Comment (jorgeorpinel) [reply content/docs/user-guide/experiment-management/index.md:50]: I guess it could be "and/or" though

Comment (jorgeorpinel) [reply content/docs/user-guide/experiment-management/index.md:50]: Hmm actually maybe no need for "isolate" mention here.

File: content/docs/user-guide/experiment-management/sharing-experiments.md [hunk 4]
```diff
@@ -1,37 +1,32 @@
...
+GitLab. Small files like experimental code and
+[DVC metafiles](/doc/user-guide/project-structure) files will go there.
+[DVC remotes](/doc/command-reference/remote) on the other hand are data storage
+locations (e.g. Amazon S3 or Google Drive). You can use them to back up and
+[share data](/doc/use-cases/sharing-data-and-model-files) files and directories
+that don't fit inside Git repos.
+
+> See this [Git remotes guide] and `dvc remote add` for info. on setting them
+> up.  
+> ⚠️ Note that only [SSH Git URLs] support DVC experiment sharing.
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/sharing-experiments.md:15]: > we should point out that dvc exp push/dvc exp pull only work with SSH

Good call @daavoo. Noted!

Comment (jorgeorpinel) [reply content/docs/user-guide/experiment-management/sharing-experiments.md:15]: p.s. do you know if there's an issue to update this info in the cmd ref? Cc @pmrowla @efiop if someone wouldn't mind doing that pls 😁

File: content/docs/user-guide/experiment-management/sharing-experiments.md [hunk 5]
```diff
@@ -1,39 +1,34 @@
...
-```
-
-Similarly, you can see the DVC remotes in you project using `dvc remote list`:
+origin  git@github.com:iterative/get-started-experiments.git (fetch)
+origin  git@github.com:iterative/get-started-experiments.git (push)
 
-```dvc
 $ dvc remote list
-storage https://remote.dvc.org/get-started-experiments
+storage s3://mybucket/my-dvc-store
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/sharing-experiments.md:31]: p.s. I also changed the DVC remote from HTTP. I don't think you can push to an HTTP remote. Cc @iesahin

**Comment by jorgeorpinel** [comment]
Looks like restyled is just not picking up broken files recently... I had to run `yarn format` on these manually:

https://app.circleci.com/pipelines/github/iterative/dvc.org/10086/workflows/9b1bf89f-a432-49f2-9a20-72fe77dd4102/jobs/10145

File: content/docs/user-guide/experiment-management/sharing-experiments.md (hunk 4 above)
Comment (daavoo) [reply content/docs/user-guide/experiment-management/sharing-experiments.md:15]: https://github.com/iterative/dvc.org/pull/2731

Comment (jorgeorpinel) [reply content/docs/user-guide/experiment-management/sharing-experiments.md:15]: Thanks @daavoo

File: content/docs/user-guide/experiment-management/sharing-experiments.md [hunk 6]
```diff
@@ -1,39 +1,33 @@
...
+GitLab. Small files like experimental code and
+[DVC metafiles](/doc/user-guide/project-structure) files will go there.
+[DVC remotes](/doc/command-reference/remote) on the other hand are data storage
+locations (e.g. Amazon S3 or Google Drive). You can use them to back up and
+[share data](/doc/use-cases/sharing-data-and-model-files) files and directories
+that don't fit inside Git repos.
+
+> See this [Git remotes guide] and `dvc remote add` for info. on setting them
+> up.  
+> ⚠️ Note that only [SSH Git URLs] support DVC experiment sharing.
```
Comment (jorgeorpinel) [inline content/docs/user-guide/experiment-management/sharing-experiments.md:15]: Hmmm so this is still not deployed. I guess `master` hasn't been deploying for a week or so now? Cc @rogermparent yelp (will ping on Slack)

Comment (jorgeorpinel) [reply content/docs/user-guide/experiment-management/sharing-experiments.md:15]: Dah never mind, this wasn't merged to `master` 🤦 false alarm

**Pull request merged**